buildSrc/src/main/groovy/org/elasticsearch/gradle/precommit/JarHellTask.groovy

```diff
@@ -39,6 +40,9 @@ public class JarHellTask extends LoggedExec {
     public JarHellTask() {
         project.afterEvaluate {
             FileCollection classpath = project.sourceSets.test.runtimeClasspath
+            if (project.plugins.hasPlugin(ShadowPlugin)) {
+                classpath += project.configurations.bundle
```

Comment:
Since `compile` extends from `bundle`, shouldn't this be in the test runtime classpath already?

Reply:
`compile` only extends from `bundle` when we're configuring and IDE.